src/EditorFeatures/CSharpTest/Diagnostics/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -192,7 +192,7 @@ public void TestConflictingProperties()
         {
             Test(
 @"class Test : [|I1|] { int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ",
-@"using System ; class Test : I1 { int I1 . Prop { get { throw new NotImplementedException ( ) ; } set { throw new NotImplementedException ( ) ; } } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
+@"class Test : I1 { int I1 . Prop { get ; set ; } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
```

Comment:
I'd much rather the languages behave consistency, with options to let users opt into differing behavior (*if* we think there's enough value in different defaults).  For auto-props I think there is *enough* value.  It feels somewhat archaic for us to so often to be using older forms in the language when we could be moving to more modern and (IMO, more pleasant) constructs.